base/promotion.jl

```diff
@@ -405,6 +405,8 @@ muladd(x::T, y::T, z::T) where {T<:Number} = x*y+z
 (&)(x::T, y::T) where {T<:Integer} = no_op_err("&", T)
 (|)(x::T, y::T) where {T<:Integer} = no_op_err("|", T)
 xor(x::T, y::T) where {T<:Integer} = no_op_err("xor", T)
+nand(x::T, y::T) where {T<:Integer} = no_op_err("nand", T)
+nor(x::T, y::T) where {T<:Integer} = no_op_err("nor", T)
```

Comment:
This should not be an error. The idiomatic Julia solution is to use duck-typing, so that `nand`/`nor` just work if a type defines `~` and `&`/`|`.

Reply:
Got it. I'll remove these two lines.